### PR TITLE
Add logging for loading in dbSNP in CollectVariantCallingMetrics.

### PR DESCRIPTION
**Pull request by nh13**
This can be quite slow for some DBSNP files, so it is nice to see some
progress.

### CODE REVIEW

File: src/main/java/picard/util/DbSnpBitSetUtil.java [hunk 1]
```diff
@@ -112,16 +118,18 @@ public static DbSnpBitSets createSnpAndIndelBitSets(final File dbSnpFile,
...
 
     /** Private helper method to read through the VCF and create one or more bit sets. */
     private static void loadVcf(final File dbSnpFile,
                                 final SAMSequenceDictionary sequenceDictionary,
                                 final Map<DbSnpBitSetUtil, Set<VariantType>> bitSetsToVariantTypes,
-                                final IntervalList intervals) {
+                                final IntervalList intervals,
+                                final Optional<Log> log) {
 
+        final Optional<ProgressLogger> progress = log.map(l -> new ProgressLogger(l, (int) 1e5, "Read", "variants"));
```
Comment (yfarjoun) [inline src/main/java/picard/util/DbSnpBitSetUtil.java]: this is so scala-esque!

Comment (nh13) [inline src/main/java/picard/util/DbSnpBitSetUtil.java]: Thanks!

**Comment by yfarjoun** [comment]
-looks good @nh13 
-any reason no to add logging for the other programs that load dbSNP: OxoG and SequencingArtifacts?

**Comment by yfarjoun** [comment]
I mean to say: 👍 but would be nice if you extended this to the other CLPs that load dbSNP.

**Pull request merged**